Documentation/general/cli-ux-guidelines.md

```diff
@@ -0,0 +1,157 @@
+.NET Core Command-Line Tools UX Guidelines
+-------------------------------------------
+
+This document outlines the User Experience (UX) of the .NET Core command lline tools (CLI).These guideliens are intended for anyone that wants to add a new command to the CLI. 

```

Comment:
add missing space between the two sentences and fix typos (guideliens and lline)